src/plugins/conditionals/README.md

```diff
@@ -113,9 +113,9 @@ EOF
 echo "key = false" > `kdb file /examples/conditionals/sub`
 
 # should fail and yield an error
-kdb export /examples/conditionals ini
-#> sub/key = false
+kdb export /examples/conditionals simpleini
```

Comment:
We should avoid simpleini, it currently does not work on macOS. (In particular such a change is strange in a PR attempting to do macOS fixes :laughing:) Any reason why you used it here?